core-tests/test-everything.sh

```diff
@@ -13,21 +13,20 @@ while getopts 'ci' flag; do
   esac
 done
 
-if [ "$force_reinstall" = "true" ] && [ -d "bower_components" ]; then
+if [ "$force_reinstall" = "true" ] && [ -d ".psc-package" ]; then
   echo "Reinstalling core packages..."
-  rm -r bower_components
+  rm -rf .psc-package
 fi
 
-# todo : fix this once core libraries reach 1.0
-yes 1 | bower i
+psc-package update
 
 if [ "$force_recompile" = "true" ] && [ -d "output" ]; then
   echo "Recompiling..."
   rm -r output
 fi
 
-stack exec purs compile 'tests/**/*.purs' 'bower_components/purescript-*/src/**/*.purs'
+stack exec purs compile 'tests/**/*.purs' '.psc-package/psc-0.11.1/*/*/src/**/*.purs'
 
-stack exec purs docs 'bower_components/purescript-*/src/**/*.purs' > core-docs.md
+stack exec purs docs '.psc-package/psc-0.11.1/*/*/src/**/*.purs' > core-docs.md
```

Comment:
I would really like to get that to work, but my bash-fu is weak, and I've tried every permutation of `${ $( $[ $@ xargamafoobar` possible to try to get it to work, and it always fails to find the Prelude.